data/schema/units/modifications.cfg

```diff
@@ -225,4 +225,7 @@
 	max=infinite
 	super="units/$modifications/base"
 	{SIMPLE_KEY duration object_duration}
+	{SIMPLE_KEY silent bool}
+	{SIMPLE_KEY armor_color_shift bool}
+	{FILTER_TAG "filter" unit ()}
```

Comment:
`{ANY_KEY string}` seems to work.  `silent` and `[filter]` are actually already defined for `[object]` in actionwml.cfg, so it seems like those aren't actually doing anything?

Reply:
I think only the ActionWML version needs `silent`. It doesn't hurt to leave it in the ModificationsWML version, but it doesn't actually do anything there. It was probably only included here because people have a tendency to add `silent=yes` even when it's not needed, in which case the `ANY_KEY` macro renders it irrelevant. The same goes for `[filter]`, as it does nothing in ModificationWML `[object]`.

Reply:
[The current output, for reference.](https://pastebin.com/51QAky0N)

For EI, it's due to its use of the TEAM_COLOR_OVERRIDE macro though, which is why I added `filter`.

Reply:
I'm not quite sure how `TEAM_COLOR_OVERRIDE` is meant to be used... if the intention is that it be valid in an event, then it probably needs the filter. So I guess you can leave the filter tag here; but you won't need the silent tag now that you have `ANY_KEY`.

(You _could_ use `ANY_TAG` instead of the filter tag though.)